x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/50_search_application_list.yml

```diff
@@ -59,79 +59,89 @@ teardown:
 "List Search Applications":
   - do:
       search_application.list: { }
+  - set: { results.0.updated_at_millis: updatedAtMillis0 }
+  - set: { results.1.updated_at_millis: updatedAtMillis1 }
+  - set: { results.2.updated_at_millis: updatedAtMillis2 }
 
   - match: { count: 3 }
 
   # Alphabetical order for results
   - match: { results.0.name: "another-test-search-application" }
   - match: { results.0.indices: [ "test-index1", "test-index2" ] }
   - match: { results.0.analytics_collection_name: "test-another-analytics" }
-  - match: { results.0.updated_at_millis: null }
+  - match: { results.0.updated_at_millis: $updatedAtMillis0 }
```

Comment:
I thought this might be sufficient because: 

1. Verifies the value is returned
2. Verifies the value isn't updated (say by always using current timestamp instead of pulling the value). 

Looking at the yaml schema, it looks like we lean very heavily on matches and I didn't see a lot of great examples for conditionals. We can certainly keep looking into it if you think we should though.

Reply:
If the value returned for `updated_at_millis` is `null`, I think the tests would pass as well, wouldn't they? That's the main issue I have, I think we should explicitly test for a > 0 value.

Reply:
Fair enough, added.